install/install.ps1

```diff
@@ -8,87 +8,196 @@ $script:ErrorActionPreference='Stop'
 # We don't currently have good cross-platform options for determining the latest release version, so we require
 # knowledge of the specific target version, which the user should set as an environment variable.
 $version = $Env:AZUREAUTH_VERSION
+#$version = "0.3.1"

```

Comment:
This looks like an accidental commit that we wont want to keep around. Note that VS Code has a pretty nice interface for staging hunks (a chunk of code) within a file, line by line - which lets you not commit testing lines you might have locally.